pygeoapi/admin.py

```diff
@@ -316,10 +317,8 @@ def get_resources(
 
     if request.format == F_HTML:
         content = render_j2_template(
-            admin.config,
-            'admin/index.html',
-            cfg['resources'],
-            request.locale,
+            admin.config, admin.config['server']['templates'],
```

Comment:
We want to pass `admin.tpl_config` to the j2 template. `pygeoapi.api.API.tpl_config` applies the `server.api_rules` to the `server.url` so the HTML templates can correctly use the pygeoapi base url.

```suggestion
            admin.tpl_config, admin.config['server']['templates'],
```